railties/lib/rails/generators/rails/app/templates/dockerignore.tt

```diff
@@ -47,6 +47,12 @@
 /.github
 <% end -%>
 
+<% unless options.skip_kamal? -%>
+# Ignore Kamal files.
+/config/deploy.yml
```

Comment:
Should it be something like this so it also ignores kamal destinations like `config/deploy.staging.yml`?

```suggestion
/config/deploy*.yml
```